backend/src/graph-info/graph-info.service.ts

```diff
@@ -0,0 +1,30 @@
+import { Injectable } from '@nestjs/common';
+import { Neo4jService } from 'nest-neo4j/dist';
+import { parseNeo4jEntityInfo } from './parseNeo4jEntityInfo';
+import { EdgeType } from '../shared/graph-information/EdgeType';
+import { NodeType } from '../shared/graph-information/NodeType';
+
+@Injectable()
+export class GraphInfoService {
+  constructor(private readonly neo4jService: Neo4jService) {}
+
+  /**
+   * Returns information about all edges of a graph
```

Comment:
maybe say "queries information about ...." and then as @returns "information about ..." since the method queries the info and then returns it